theano/sandbox/cuda/tests/test_nnet.py

```diff
@@ -21,6 +21,7 @@
 else:
     mode_with_gpu = theano.compile.mode.get_default_mode().including('gpu')
     mode_without_gpu = theano.compile.mode.get_default_mode()
+    mode_without_gpu = mode_without_gpu.excluding('gpu')
```

Comment:
I don't know if it fits and It's not important at all but maybe `mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpu')` would be clearer.